libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -534,7 +539,7 @@ export const mapEstateInfo = (syslaData: DanarbuUpplRadstofun): EstateInfo => {
       ? new Date(syslaData.danardagur)
       : new Date(),
     districtCommissionerHasWill: Boolean(syslaData?.erfdaskra),
-    knowledgeOfOtherWills: syslaData.erfdakraVitneskja ? 'Yes' : 'No',
+    knowledgeOfOtherWills: syslaData.erfdakraVitneskja ? 'yes' : 'no',
```

Comment:
better to put into a constant and reuse that constant instead of redeclaring case sensitive strings